roles/rabbitmq/tasks/main.yml

```diff
@@ -15,3 +15,14 @@
     enabled: yes
     state: started
   tags: rabbitmq
+
+- name: Add RabbitMQ plugins
+  become: yes
+  rabbitmq_plugin:
+    names: "{{ rabbitmq_plugins }}"
+    state: enabled
+    new_only: yes
```

Comment:
> Also what will happen if on first deployment we have rabbitmq_plugins: [rabbitmq_management] and on second deployment we remove it: rabbitmq_plugins: [].
Will that automatically disable unneeded plugins for us?

With "new_only", it will only ever add new plugins. It won't remove old ones. I could add a `rabbitmq_remove_unlisted_plugins` var (unsure on that verbose name) that would do `new_only: no` when specifically requested, but default to `yes`. Maybe it would be clearer to use `rabbitmq_plugins_new_only` to merely pass through the bool, but default it to yes.

Reply:
done. I used the simpler `rabbitmq_plugins_new_only` var.